wordpress-mobile/WordPress-iOS
# Include locale specifier in WPCom API requests

Closes #3780 

To test:

- First launch the app as normal and generate some known error messages from the API, or perhaps breaking some request params in code.
- Change your simulator/device language in Settings > General > Language and Region > iPhone Language.
- Generate errors again, ensure the errors display in the selected language.

Note: Not all iOS languages are supported via WordPress.com, which will default itself to "en".

Needs review: @astralbodies
